cmd/dep/godep_importer.go

```diff
@@ -17,7 +17,10 @@ import (
 	"github.com/pkg/errors"
 )
 
-const godepJSONName = "Godeps.json"
+const (
```

Comment:
Hmm, I don't think it will - it uses the correct slash when _joining_ arguments, but it does not go through and replace slashes within arguments. That's `filepath.FromSlash()`.